README.md

```diff
@@ -1,76 +1,76 @@
+The following will walk you through setting up the new hydra head.  Understand that there will be manual tasks that must be done outside of this before you know if everything worked properly.
 
 ## Copy and Rename
 
-Copy and rename this directory from git to your desktop.  Name it `hydra-#{projectname}` for consistancy.  
+Copy and rename this directory from git to your desktop.  Name it `hydra-#{projectname}` for consistancy.
 
 ## REMOVE GIT
 
-This is an important step.  In creating new hydra repos, we don't want to over-ride this current repo.  
+This is an important step.  In creating new hydra repos, we don't want to over-ride this current repo.
 - Be sure you are in the root of this github repo and type `sudo rm -rf .git`
-- Alternately if you can see hidden folders you can use your GUI to look up the `.git` directory and delete it manually.  
+- Alternately if you can see hidden folders you can use your GUI to look up the `.git` directory and delete it manually.
 
 ## SETUP FILE
-Using the terminal change directory to wvu_setup directory.  From this directory you will see a setup.rb file.  Open this file and change the information for your specific repository.   
+Using the terminal change directory to wvu_setup directory.  From this directory you will see a setup.rb file.  Open this file and change the information for your specific repository.
 
 ### BASIC SETUP
 
- - 'TestModelName' :: Change this to your model name using CamelCase writing.  
- - 'tdna' :: Change this to your project abbreviation or name that is often generated from the MFCS export.  
+ - 'TestModelName' :: Change this to your model name using CamelCase writing.
+ - 'tdna' :: Change this to your project abbreviation or name that is often generated from the MFCS export.
  - 'test collection name' :: Change this to whatever name you want your collection to be.
  - **ALL OF THESE ARE CASE SENSITIVE**
 
 ## Executing the File
 
 - The system requirements for each system may be different.  The file was given executable permissions, make sure this has carried over to your system.  Also make sure that the first line reads as
-`#!/usr/bin/env ruby`.  This ensures the the file will be run as a ruby file and an executable.  
+`#!/usr/bin/env ruby`.  This ensures the the file will be run as a ruby file and an executable.
 - `cd wvu_setup`
 - `ruby setup.rb`
 
 ## After File Executes
 - Delete wvu-setup directory
 - Verify that Git was Deleted
-- Setup the New Github Link 
+- Setup the New Github Link
 
-## Hydra Head Manual Setups 
-There isn't any way to script all of this without lots of pre-planning and forcing hydra to adhear to a certain standard. 
+## Hydra Head Manual Setups
+There isn't any way to script all of this without lots of pre-planning and forcing hydra to adhear to a certain standard.
```

Comment:
```suggestion
There isn't any way to script all of this without lots of pre-planning and forcing hydra to adhere to a certain standard.
```

Reply:
I know this was preexisting, but since we're already making changes to this file... 🙃